CONTRIBUTING.md

```diff
@@ -0,0 +1,235 @@
+
+You should also read our [coding guide](https://github.com/influxdata/influxdb/blob/master/CODING_GUIDELINES.md), to understand better how to write code for InfluxDB.
+
+Submitting a pull request
+------------
+To submit a pull request you should fork the InfluxDB repository, and make your change on a feature branch of your fork. Then generate a pull request from your branch against *master* of the InfluxDB repository. Include in your pull request details of your change -- the why *and* the how -- as well as the testing your performed. Also, be sure to run the test suite with your change in place. Changes that cause tests to fail cannot be merged.
+
+There will usually be some back and forth as we finalize the change, but once that completes it may be merged.
+
+To assist in review for the PR, please add the following to your pull request comment:
+
+```md
+- [ ] CHANGELOG.md updated
+- [ ] Rebased/mergable
+- [ ] Tests pass
+- [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
+```
+
+Security Vulnerability Reporting
+---------------
+InfluxData takes security and our users' trust very seriously. If you believe you have found a security issue in any of our
+open source projects, please responsibly disclose it by contacting security@influxdata.com. More details about 
+security vulnerability reporting, 
+including our GPG key, [can be found here](https://www.influxdata.com/how-to-report-security-vulnerabilities/).
+
+
+Signing the CLA
+---------------
+
+If you are going to be contributing back to InfluxDB please take a
+second to sign our CLA, which can be found
+[on our website](https://influxdata.com/community/cla/).
+
+Installing Go
+-------------
+InfluxDB requires Go 1.12.
+
+At InfluxData we find gvm, a Go version manager, useful for installing Go. For instructions
+on how to install it see [the gvm page on github](https://github.com/moovweb/gvm).
+
+After installing gvm you can install and set the default go version by
+running the following:
+
+    gvm install go1.11
+    gvm use go1.12 --default
+
+Revision Control Systems
+-------------
+Go has the ability to import remote packages via revision control systems with the `go get` command.  To ensure that you can retrieve any remote package, be sure to install the following rcs software to your system.
+Currently the project only depends on `git` and `mercurial`.
+
+* [Install Git](http://git-scm.com/book/en/Getting-Started-Installing-Git)
+* [Install Mercurial](http://mercurial.selenic.com/wiki/Download)
+
+Getting the source
+------
+Setup the project structure and fetch the repo like so:
+
+```bash
+    cd $GOPATH/src
+    git clone git@github.com:influxdata/influxdb.git
+    cd influxdb
+    export  GO111MODULE=on
+    go mod vendor
+    go install ./...
```

Comment:
ignores rust and typescript.

Multiple languages requires our Makefile system

Reply:
Need someone to write this.

Reply:
In the interests of getting this merged I'm resolving this comment.  We can make iterative changes going forwards.

Reply:
Would you add an issue into the backlog?  That way we won't lose context. Thanks!

Reply:
Done.   https://github.com/influxdata/influxdb/issues/16302
I'll follow up with the right people to get that written.